profiling/src/profile/profiled_endpoints.rs

```diff
@@ -0,0 +1,53 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2021-Present Datadog, Inc.
+
+use std::ops::AddAssign;
+
+use indexmap::IndexMap;
+use serde::ser::SerializeMap;
+use serde::Serialize;
+
+#[derive(Default, PartialEq, Eq, Debug, Clone)]
+pub struct ProfiledEndpointsStats {
+    count: IndexMap<String, i64>,
```

Comment:
This could be a regular HashMap, yeah? You don't use it by index anywhere, as far as I can tell?

Reply:
✔️